packages/cli/generators/project/templates/.vscode/launch.json

```diff
@@ -0,0 +1,55 @@
+{
+  "configurations": [
```

Comment:
I think we need to add the launch program for starting the application, i.e. something like this:
```
{
  // Use IntelliSense to learn about possible attributes.
  // Hover to view descriptions of existing attributes.
  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
  "version": "0.2.0",
  "configurations": [
    {
      "type": "node",
      "request": "launch",
      "name": "Launch Program",
      "skipFiles": [
        "<node_internals>/**"
      ],
      "program": "${workspaceFolder}/index.js",
    }
  ]
}

```